packages/launcher/static/images/icons/right-arrow-grey.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you process this file with SVGO ([here is an online version](https://jakearchibald.github.io/svgomg/)) to remove the comments and potentially simplify the elements please?